code/edgeidentity/build.gradle

```diff
@@ -73,16 +76,16 @@ android {
     }
 }
 
```

Comment:
Let add debuggable true to
buildTypes {
        debug {
            testCoverageEnabled true
            debuggable true
        }